src/functions.php

```diff
@@ -35,7 +35,8 @@ function is_zipkin_timestamp($timestamp)
      * @return bool
      */
     function is_zipkin_trace_identifier($identifier) {
-        return ctype_xdigit((string) $identifier) && strlen((string) $identifier) === 32;
```

Comment:
What is TraceSpanIdentifier? If you're referring to TraceIdentifier, it will be supported with this change.